nrf70_bm_lib/include/nrf70_bm_lib.h

```diff
@@ -144,7 +144,7 @@ struct nrf70_version {
 
 /** @brief Wi-Fi scanning types. */
 enum nrf70_scan_type {
-	/** Active scanning (default). */
+	/** Active scanning (default) - two probe requests are sent on each channel. */
```

Comment:
Should we add this in the main scan API docs as well? It's fine here of course.

Reply:
Same for the rest of the "default" parameters.

Reply:
The scan API doc in NCS already has these. https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/protocols/wifi/scan_mode/scan_operation.html

Reply:
I meant the fact that there are 2 probes sent out. Nevermind, I'll raise this comment in the appropriate place.